### PR TITLE
Replace secureimage by reCaptcha

### PR DESCRIPTION
**Pull request by kumy**

### CODE REVIEW

**Comment by boly38** [comment]
how to manual launch QA (while waiting for automated step) :

image

**Comment by boly38** [comment]
reminder

please ensure that there is an unique ID on the reCaptcha html element
create a new issue in geokrety website project to add a new test about registration basic structure

File: website/templates/konfig.php [hunk 1]
```diff
@@ -10,6 +10,10 @@
 // Google map Api key
 $GOOGLE_MAP_KEY = 'xxx';
 
+// reCaptcha Api key
+$GOOGLE_RECAPTCHA_PUBLIC_KEY = 'xxx';
```
[the remark this answers is not in the document]
Comment (kumy) [reply website/templates/konfig.php]: Done!

[the remark this answers is not in the document]
Comment (kumy) [reply website/templates/konfig.php]: What should we do on `rec` and `boly38` instances ?

**Comment by kumy** [comment]
@boly38 PR updated

**Pull request merged**